CMakeLists.txt

```diff
@@ -555,11 +555,6 @@ if(WIN32)
     set(CMAKE_EXE_LINKER_FLAGS_RELEASE "${CMAKE_EXE_LINKER_FLAGS_RELEASE} /LTCG /IGNORE:4075 /DEBUG /OPT:REF /OPT:ICF")
 
     # Find the right PGO file.
-    if(QUIC_USE_XDP)
-        set(QUIC_PGO_NAME "msquic.${QUIC_TLS}.xdp.pgd")
-        set(QUIC_PGO_FILE "${CMAKE_CURRENT_SOURCE_DIR}/src/bin/winuser/pgo_${SYSTEM_PROCESSOR}/${QUIC_PGO_NAME}")
-    endif()
```

Comment:
This will make things a bit complicated, because we'll need to run both XDP and non-XDP and combine their PGO files.

Reply:
I'm not familiar with PGO, How to combine these two?

Reply:
We will need to combine the perf testing scenarios to test both normal and XDP. We can do that in a follow up PR.

Reply:
Opened https://github.com/microsoft/msquic/issues/3866 to track.